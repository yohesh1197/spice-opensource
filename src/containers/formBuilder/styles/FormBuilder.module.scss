@import '../../../styles/variables';

.fieldContainer {
  background-color: var(--white);
  border-radius: 8px;
  margin-bottom: 1rem;
}

.body {
  height: calc(100vh - #{$app-navbar-height} - 7.375em);
}

.error {
  padding-left: 1px;
  padding-top: 1px;
  font-size: 0.85rem;
  min-height: 1.5rem;
  color: $field-error-color;
}

.danger {
  border: 1px solid $field-error-color !important;
}

label {
  margin-bottom: 0.5rem;
  color: $field-label-color;
  line-height: 1.25rem;
  font-size: 0.875rem;
  padding-left: 1px;
}

.optionsList {
  display: table;
  width: 100%;
}

.editBtnImg {
  filter: brightness(0) invert(1);
  margin-right: 0.4375rem;
}

.sideMenu {
  width: 240px;
  background-color: var(--white);
  border-radius: 0.75rem;
  border-style: solid;
  border-color: $side-menu-border;
  border-width: 1px;
  max-height: calc(100vh - #{$app-navbar-height} - 7.375em - 2.875rem);
  overflow: auto;
  .menuItem {
    border-inline-start: 0.25rem solid var(--white);
    &:hover {
      border-inline-start-color: $menu-item-hover-background;
      background-color: $menu-item-hover-background;
      color: $menu-item-active-color;
    }
    &:active {
      color: var(--white);
      background-color: $menu-item-active-color;
      border-inline-start-color: $menu-item-active-color;
    }
    &.selected {
      font-weight: 700;
      pointer-events: none;
      border-inline-start: 0.25rem solid $menu-item-active-color !important;
      color: $menu-item-active-color;
    }
  }
}

.addNew {
  color: $menu-item-active-color;
  border-style: solid;
  border-color: $side-menu-border;
  border-width: 1px;
  border-radius: 0.75rem;
  &:hover {
    background-color: $menu-item-hover-background;
    color: $menu-item-active-color;
  }
}
.input {
  width: 100%;
}
.tagInput {
  max-height: 120px;
  min-height: 92px;
  .taginputContainer {
    border: 1px solid $field-border-color;
    padding: 5px;
    border-radius: 5px;
    max-height: 100px;
    overflow-y: auto;
    .items {
      display: inline-block;
      padding: 2px 5px;
      border-radius: 0.25rem;
      box-sizing: border-box;
      margin: 2px;
      cursor: pointer;
      background-color: $field-deactivate-color;
      font-size: 0.875rem;
    }
    .disabledItems {
      border: 1px solid $field-border-color;
    }

    input {
      outline: none;
      border: none;
      font-size: 0.875rem;
    }
  }
}
.disabledContainer {
  cursor: not-allowed;
  background-color: $field-deactivate-color;
  &:hover {
    border-color: $field-border-color;
  }
}
.remove {
  font-weight: bold;
  &:hover {
    color: $field-error-color;
  }
}
.label {
  margin-bottom: 0.5rem;
  color: $field-label-color;
  line-height: 1.25rem;
  padding-left: 1px;
  font-size: 0.875rem;
}

.formBuilderViewTwo {
  .viewContent {
    border-radius: 12px;
  }
  .deleteIcon {
    right: 6.5rem;
    bottom: 1rem;
    transform: translateX(50%);
  }
  .addNewItem {
    color: var(--theme-color);
    line-height: 1.25rem;
    cursor: pointer;
    display: flex;
    align-items: center;
    width: auto;
    margin-right: 1rem;
  }
}
.scroll {
  overflow-y: auto;
  max-height: 60vh;
}
.customizationFont {
  font-size: 0.875rem;
}

.conditionsContainer {
  border: 1px solid $field-border-color;
  border-radius: 10px;
  padding-top: 1rem;
}

.actionIcons {
  width: 5rem;
  justify-content: flex-start;
}

.trueLabel {
  margin-right: 10px !important;
}

.accWorkflowBtns {
  display: flex;
  height: 2.5rem;
}

.cultureDropdown {
  width: 13.5rem;
}
