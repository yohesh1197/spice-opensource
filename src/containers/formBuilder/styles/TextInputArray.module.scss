@import '../../../styles/variables';

.textInputArray {
  label {
    margin-bottom: 0.5rem;
    color: $field-label-color;
    line-height: 1.25rem;
    padding-left: 1px;
  }
  .borderOnNoText {
    box-shadow: inset 0px 4px 8px $field-focus-box-shadow-color;
    border: 0.5px solid $field-border-color !important;
    margin: 0.25rem;
  }
  .input {
    border: 1px solid transparent;
    border-radius: 0.187rem;
    padding: 0.625rem 0.55rem 0.75rem;
    width: 100%;
    color: $field-text-color;
    justify-content: space-between;
    max-height: 1.25rem;
    max-height: 6rem;
    overflow: auto;
    &:focus {
      box-shadow: inset 0px 4px 8px $field-focus-box-shadow-color;
      border: 0.5px solid $field-border-color !important;
      outline: none;
    }
    &::-webkit-input-placeholder {
      /* Chrome/Opera/Safari */
      color: $field-placeholder-color;
    }
    &::-moz-placeholder {
      /* Firefox 19+ */
      color: $field-placeholder-color;
    }
    &:-ms-input-placeholder {
      /* IE 10+ */
      color: $field-placeholder-color;
    }
    &:-moz-placeholder {
      /* Firefox 18- */
      color: $field-placeholder-color;
    }
    &:disabled {
      cursor: not-allowed;
      background-color: $field-deactivate-color;
      &:hover {
        border-color: $field-border-color;
      }
    }
  }
}
