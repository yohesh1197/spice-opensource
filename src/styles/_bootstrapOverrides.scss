@import './variables';

// Bootstrap overrides

// theme overrides
$primary: #2514be;
$success: #00a372;
$danger: #e31b28;
$light: #fff;

// spacing overrides
$spacer: 1rem;
$spacers: (
  0: 0,
  1: $spacer * 1,
  2: $spacer * 2,
  3: $spacer * 3,
  4: $spacer * 4,
  5: $spacer * 5,
  6: $spacer * 6,
  7: $spacer * 7,
  8: $spacer * 8,
  9: $spacer * 9,
  10: $spacer * 10,
  // special cases
  '0dot125': $spacer * 0.125,
  '0dot25': $spacer * 0.25,
  '0dot375': $spacer * 0.375,
  '0dot5': $spacer * 0.5,
  '0dot625': $spacer * 0.625,
  '0dot75': $spacer * 0.75,
  '0dot875': $spacer * 0.875,
  '1dot125': $spacer * 1.125,
  '1dot25': $spacer * 1.25,
  '1dot375': $spacer * 1.375,
  '1dot5': $spacer * 1.5,
  '1dot75': $spacer * 1.75,
  '1dot875': $spacer * 1.875,
  '2dot25': $spacer * 2.25,
  '2dot5': $spacer * 2.5,
  '2dot75': $spacer * 2.75,
  '3dot125': $spacer * 3.125,
  '3dot75': $spacer * 3.75,
  // negative
  neg-1dot25: -$spacer * 1.25,
  neg-2dot75: -$spacer * 2.75
);

$grid-gutter-width: 1.5rem;
$gutters: (
  0: 0,
  1: $spacer * 1,
  2: $spacer * 2,
  3: $spacer,
  4: $spacer * 3,
  5: $spacer * 4,
  // special cases
  '0dot625': $spacer * 0.625,
  '1dot25': $spacer * 1.25
);

@import 'bootstrap/scss/bootstrap';

.dropdown-menu {
  border: 0.3px solid $dropdown-border-color;
  box-shadow: 0px 4px 8px $dropdown-box-shadow;
  overflow: hidden;
  border-radius: 0.5rem;
}

.dropdown-toggle {
  &.show {
    img {
      transform: rotate(180deg);
    }
    &.light {
      background-color: $dropdown-toggler-light-hover-color;
    }
    &.primary-btn {
      background-color: var(--primary-btn-active-background);
    }
  }
  &::after {
    display: none !important;
  }
}

.dropdown-item {
  padding: 0.625rem 1rem !important;
  &:hover {
    background-color: $dropdown-item-hover-background-color !important;
    color: $dropdown-item-hover-color !important;
  }
}

.pagination-dropdown {
  &.dropdown-menu {
    min-width: unset !important;
  }
}

.modal-footer > * {
  margin: 0 !important;
}

.accordion-button {
  color: var(--primary-title-text-color) !important;
  &:focus {
    border-color: none !important;
    box-shadow: none !important;
  }
}
.accordion {
  .accordion-item {
    border: 1px solid $field-border-color;
    &:last-of-type {
      border-bottom-right-radius: 12px;
      border-bottom-left-radius: 12px;
    }
    &:first-of-type {
      border-top-left-radius: 12px;
      border-top-right-radius: 12px;
      .accordion-button {
        border-top-left-radius: 12px;
        border-top-right-radius: 12px;
        border-bottom-right-radius: 12px;
        border-bottom-left-radius: 12px;
      }
    }
  }
}
