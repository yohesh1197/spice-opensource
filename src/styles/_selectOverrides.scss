@import './variables';
@import './theme';

.select-field__menu--generic {
  @extend .highlight-small-text;
  border: 0.3px solid $dropdown-border-color !important;
  box-shadow: 0px 4px 8px $dropdown-box-shadow !important;
  overflow: hidden !important;
  border-radius: 0.5rem !important;
  .select-field__option {
    padding: 0.625rem 1rem !important;
    &:hover:not(.select-field__option--is-selected) {
      background-color: $dropdown-item-hover-background-color !important;
      color: $dropdown-item-hover-color !important;
    }
    &.select-field__option--is-focused:not(.select-field__option--is-selected) {
      color: $dropdown-item-hover-color !important;
    }
    &.select-field__option--is-selected {
      background-color: var(--medium-blue) !important;
      color: var(--white) !important;
    }
  }
}
.select-field {
  .select-field__control {
    min-height: 2.5rem !important;
    border-color: $field-border-color !important;
    &:hover {
      border-color: $field-focus-border-color !important;
    }
    &.select-field__control--is-focused {
      border-color: $field-focus-border-color !important;
      box-shadow: inset 0px 4px 8px $field-focus-box-shadow-color !important;
    }
    &.select-field__control--is-disabled {
      pointer-events: auto;
      cursor: not-allowed;
      background-color: $field-deactivate-color;
      &:hover {
        border-color: $field-border-color;
      }
    }
    .select-field__input-container .select-field__input,
    .select-field__single-value {
      color: $field-text-color !important;
      font-size: 0.875rem !important;
    }
    .select-field__indicator-separator {
      display: none !important;
    }
    .select-field__placeholder {
      font-size: 0.875rem !important;
      color: $field-placeholder-color;
    }
  }
  &.danger {
    .select-field__control--is-focused,
    .select-field__control {
      border-color: $field-error-color !important;
    }
  }
  .select-field__menu {
    @extend .select-field__menu--generic;
  }
}
.select-field__menu-portal {
  .select-field__menu {
    @extend .select-field__menu--generic;
  }
}

.pagination-select {
  @extend .select-field;
  .select-field__control {
    min-height: 2.125rem !important;
    min-width: 4rem !important;
  }
  .select-field__indicator {
    padding: 5px 4px !important;
  }
  .select-field__input-container {
    margin-bottom: 1px !important;
  }
  .select-field__menu {
    width: unset !important;
  }
}
