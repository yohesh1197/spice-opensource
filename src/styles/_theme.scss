:root {
  --red: #e31b28;
  --red-5: #fef4f4;
  --black: #000000;
  --black-65: #595959;
  --black-45: #8c8c8c;
  --black-30: #b3b3b3;
  --black-rgb: 0, 0, 0;
  --white: #fff;
  --medium-blue: #2514be;
  --medium-blue-3: #f8f8fd;
  --medium-blue-20: #2514be36;
  --medium-blue-60: #6165de;
  --medium-blue-120: #1e1098;
  --medium-blue-140: #160c72;
  --grey: #deddd8;
  --green: #00a372;
  --off-white: #fcfbf8;
  --burnt-orange: #c35721;
  --light-grey-blur: #dfdfdfd4;

  // text
  --primary-title-text-color: var(--black);
  --secondary-title-text-color: var(--black);

  --theme-color: var(--medium-blue);
  --danger-color: var(--red);

  //
  --highlight-color: var(--black);
  --subtle-text-color: var(--black-65);

  // background
  --primary-background-color: var(--medium-blue);

  // primary btn
  --primary-btn-background: var(--medium-blue);
  --primary-btn-hover-background: var(--medium-blue-120);
  --primary-btn-active-background: var(--medium-blue-140);
  --primary-btn-text-color: var(--white);
  --primary-btn-hover-text-color: var(--white);
  --primary-btn-active-text-color: var(--white);
  --primary-btn-outline: var(--medium-blue);

  // secondary-btn
  --secondary-btn-background: var(--white);
  --secondary-btn-hover-background: var(--medium-blue-3);
  --secondary-btn-active-background: var(--medium-blue);
  --secondary-btn-text-color: var(--medium-blue);
  --secondary-btn-active-text-color: var(--white);
  --secondary-btn-border-color: var(--medium-blue);
  --secondary-btn-active-border-color: var(--medium-blue-140);

  // load-more-btn
  --load-more-btn-background: transparent;
  --load-more-btn-hover-background: var(--medium-blue-3);
  --load-more-btn-active-background: var(--medium-blue);
  --load-more-btn-text-color: var(--medium-blue);
  --load-more-btn-active-text-color: var(--white);
  --load-more-btn-border-color: var(--white);
  --load-more-btn-active-border-color: var(--medium-blue);

  // danger-btn
  --danger-btn-background: var(--white);
  --danger-btn-hover-background: var(--red-5);
  --danger-btn-active-background: var(--red);
  --danger-btn-text-color: var(--red);
  --danger-btn-active-text-color: var(--white);
  --danger-btn-border-color: var(--red);
  --danger-btn-active-border-color: var(--red);

  // btn
  --disabled-btn-color: rgba(var(--black-rgb), 0.4);

  // switchbox
  --checkbox-selection-color: var(--medium-blue);
  --checkbox-un-selected-bar-color: var(--grey);
  --checkbox-un-selected-circle-color: var(--white);

  //error
  --primary-error-color: var(--red) !important;

  // reorder
  --drag-handle-background-color: var(--off-white);
  --reorerableItem-border-color: var(--black-30);
  --reorerableItem-background-color: var(--white);
  --box-shadow-color-when-drag: rgba(var(--black-rgb), 0.12);
}

.primary-btn {
  background-color: var(--primary-btn-background);
  color: var(--primary-btn-text-color);
  border: 1px solid var(--primary-btn-background);
  &:hover {
    background-color: var(--primary-btn-hover-background);
    color: var(--primary-btn-hover-text-color);
    border-color: var(--primary-btn-hover-background);
  }
  &:active {
    background-color: var(--primary-btn-active-background);
    color: var(--primary-btn-active-text-color);
    border-color: var(--primary-btn-active-background);
  }
  &:disabled {
    background-color: var(--disabled-btn-color);
    border-color: var(--disabled-btn-color);
  }
  &:focus {
    border-color: var(--primary-btn-outline);
  }
}

.secondary-btn {
  background-color: var(--secondary-btn-background);
  color: var(--secondary-btn-text-color);
  border: 1px solid var(--secondary-btn-border-color);
  &:hover {
    background-color: var(--secondary-btn-hover-background);
    color: var(--secondary-btn-text-color);
  }
  &:active {
    background-color: var(--secondary-btn-active-background);
    color: var(--secondary-btn-active-text-color);
    border-color: var(--secondary-btn-active-border-color);
  }
  &:disabled {
    border-color: var(--disabled-btn-color);
    color: var(--disabled-btn-color);
  }
}

.load-more-btn {
  background-color: var(--load-more-btn-background);
  color: var(--load-more-btn-text-color);
  border: 1px solid transparent;
  &:hover {
    background-color: var(--load-more-btn-hover-background);
    color: var(--load-more-btn-text-color);
  }
  &:disabled {
    border-color: var(--disabled-btn-color);
    color: var(--disabled-btn-color);
  }
}

.danger-btn {
  background-color: var(--danger-btn-background);
  color: var(--danger-btn-text-color);
  border: 1px solid var(--danger-btn-border-color);
  &:hover {
    background-color: var(--danger-btn-hover-background);
    color: var(--danger-btn-text-color);
  }
  &:active {
    background-color: var(--danger-btn-active-background);
    color: var(--danger-btn-active-text-color);
    border-color: var(--danger-btn-active-border-color);
  }
  &:disabled {
    border-color: var(--disabled-btn-color);
    color: var(--disabled-btn-color);
  }
}

.primary-title {
  color: var(--primary-title-text-color);
}

.secondary-title {
  color: (--secondary-title-text-color);
}

.primary-bg {
  background-color: var(--primary-background-color);
}

.page-title {
  font-weight: bold;
  font-size: 1.5rem;
  line-height: 2rem;
  letter-spacing: -0.02em;
  color: var(--primary-title-text-color);
}

.highlight-text {
  font-size: 1rem;
  line-height: 1.375rem;
  letter-spacing: -0.02em;
}

.highlight-small-text {
  font-weight: normal;
  font-size: 0.875rem;
  line-height: 1.25rem;
  letter-spacing: -0.02em;
}

.subtle-color {
  color: var(--subtle-text-color);
}

.subtle-small-text {
  font-size: 0.75rem;
  line-height: 0.9375rem;
  @extend .subtle-color;
}

.danger-text {
  color: var(--danger-color);
}

.theme-text {
  color: var(--theme-color);
}
