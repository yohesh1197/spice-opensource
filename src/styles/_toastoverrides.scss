$toast-message-color: var(--black);

:root {
  --toastify-toast-min-height: 46px !important;
  --toastify-color-warning: var(--red) !important;
  --toastify-icon-color-success: var(--green) !important;
  --toastify-icon-color-info: var(--medium-blue-60) !important;
  --toastify-toast-font-style: AppFont, sans-serif !important;
}

.Toastify__toast-body {
  padding: 0 !important;
  margin: 0 !important;
  font-size: 1rem !important;
  line-height: 1.375rem !important;
  align-items: flex-start !important;
}

.Toastify__toast {
  font-family: var(--toastify-toast-font-style) !important;
  padding: 0.75rem !important;
  box-shadow: 0px 8px 16px rgba(83, 86, 90, 0.2) !important;
  border-radius: 0.5rem !important;
}

.Toastify__toast--warning {
  color: var(--toastify-color-warning) !important;
}

.Toastify__toast--success {
  color: var(--toastify-icon-color-success) !important;
}

.Toastify__toast--info {
  color: var(--toastify-icon-color-info) !important;
}

.Toastify__toast-icon {
  margin-inline-end: 0.4375rem !important;
  margin-top: 0.15rem !important;
  img {
    width: 1rem !important;
    height: 1rem !important;
  }
}

.Toast_body {
  color: $toast-message-color !important;
  font-size: 1rem !important;
}
