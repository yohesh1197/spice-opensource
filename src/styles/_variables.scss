// Input field variables
$field-label-color: var(--black-65);
$field-text-color: var(--black);
$field-border-color: rgba(var(--black-rgb), 0.4);
$field-focus-border-color: var(--black-65);
$field-focus-box-shadow-color: rgba(var(--black-rgb), 0.1);
$field-placeholder-color: rgba(var(--black-rgb), 0.4);
$field-error-color: var(--red);
$field-deactivate-color: var(--off-white);

// Icons
$icon-hover-background-color: var(--medium-blue-3);

// text color
$link-text-color: var(--medium-blue);
$privacy-text-color: var(--black);

// background
$primary-background: var(--medium-blue-3);

// loader
$loader-color: var(--medium-blue);
$loader-background: var(--light-grey-blur);

// dropdown
$dropdown-toggler-light-hover-color: var(--medium-blue-3);
$dropdown-item-hover-background-color: var(--medium-blue-3);
$dropdown-item-hover-color: var(--medium-blue);
$dropdown-border-color: var(--black-30);
$dropdown-box-shadow: 0px 8px 16px rgba(83, 86, 90, 0.2);

// header height
$app-navbar-height: 4.125rem;

// main content
$main-content-max-width: 1280px;
//table
$table-head-background: var(--off-white);
$table-head-color: var(--black-45);

// side menu
$side-menu-border: var(--black-30);
$menu-item-hover-background: var(--medium-blue-3);
$menu-item-active-color: var(--medium-blue);
$menu-item-hover-color: var(--medium-blue-60);
//modal
$modal-background: rgba(var(--black-rgb), 0.8);

//divider
$divider-background-color: var(--black-30);
