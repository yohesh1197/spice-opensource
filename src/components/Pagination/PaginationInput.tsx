import React, { useCallback, useState } from 'react';
import styles from './Pagination.module.scss';

interface IInputProps extends React.InputHTMLAttributes<HTMLInputElement> {
  maxNumber: number;
  onPagination: (page: number) => void;
}

/**
 * Input that indicates the user by filling danger color in background
 * when user types in a number greater than maxNumber property
 */
const PaginationInput = ({ maxNumber, className, onChange, onPagination, ...otherProps }: IInputProps) => {
  const [error, setError] = useState(false);
  const [value, setValue] = useState('');
  const onKeyPress = useCallback(
    (e: React.KeyboardEvent<HTMLInputElement>) => {
      if (e.key === 'Enter' && Number(value) && !error) {
        setValue('');
        onPagination(Number(value));
      }
    },
    [onPagination, value, error]
  );

  const onChangeEvent = useCallback(
    (event: React.ChangeEvent<HTMLInputElement>) => {
      const { value: nxtValue } = event.target;
      if (isNaN(Number(nxtValue))) {
        event.preventDefault();
        return;
      }
      setValue(event.target.value);
      onChange?.(event);
      setError(nxtValue !== '' && (Number(nxtValue) > maxNumber || Number(nxtValue) < 1 || Number(nxtValue) % 1 !== 0));
    },
    [onChange, maxNumber]
  );
  return (
    <input
      {...otherProps}
      value={value}
      className={`${className} ${styles.goToPage} ${error ? styles.error : ''}`}
      onChange={onChangeEvent}
      onKeyPress={onKeyPress}
    />
  );
};

export default PaginationInput;
