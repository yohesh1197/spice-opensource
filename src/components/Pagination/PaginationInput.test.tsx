import React from 'react';
import { shallow } from 'enzyme';
import PaginationInput from './PaginationInput';
import styles from './Pagination.module.scss';
import APPCONSTANTS from '../../constants/appConstants';

describe('PaginationInput', () => {
  const props = {
    maxNumber: APPCONSTANTS.ROWS_PER_PAGE_OF_TABLE,
    onPagination: jest.fn(),
  };
  const component = shallow(<PaginationInput {...props} />);
  
  it('renders correctly', () => {
    expect(component).toMatchSnapshot();
  });

  it('calls onPagination when Enter key is pressed and input value is valid', () => {
    component.find('input').simulate('change', { target: { value: '5' } });
    component.find('input').simulate('keyPress', { key: 'Enter' });
    expect(props.onPagination).toHaveBeenCalledWith(5);
  });

  it('displays error class when input value is not valid', () => {
    component.find('input').simulate('change', { target: { value: '200' } });
    expect(component.find('input').hasClass(`${styles.error}`)).toBeTruthy();
  });
});
