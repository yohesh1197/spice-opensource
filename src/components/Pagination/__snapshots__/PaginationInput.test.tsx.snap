// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PaginationInput renders correctly 1`] = `ShallowWrapper {}`;
